holidays/countries/bonaire_sint_eustatius_and_saba.py

```diff
@@ -0,0 +1,82 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.groups import ChristianHolidays, InternationalHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class BonaireSintEustatiusAndSaba(HolidayBase, ChristianHolidays, InternationalHolidays):
+    """Bonaire Sint Eustatius And Saba holidays.
+
+    References:
+        * [Public Holidays 2025](https://www.sabagov.nl/residents-visitors/public-holidays)
+        * [Public Holidays in Sint Eustatius in 2025](https://www.officeholidays.com/countries/bonaire-st-eustatius-saba/sint-eustatius/2025)
+        * [Upcoming Bonaire, St Eustatius and Saba Public Holidays](https://www.qppstudio.net/public-holidays/bonaire__st_eustatius_and_saba.htm)
+    """
+
+    country = "BQ"
+    default_language = "en_BQ"
```

Comment:
Since all three parts are special municipalities of the Netherlands, it would be logical to have the Dutch as the default language. And need to support subdivisions, as they have their own holidays (Saba Day, Sint Eustatius Flag Day, Emancipation Day).